Model/SyncStatus.php

```diff
@@ -94,27 +94,18 @@ public function sync($order)
     /**
      * @param Order $order
      * @param array $charge
+     * @return void
      */
     private function markPaymentSuccessful($order, $charge)
     {
-        $refunded_amount = isset($charge['refunded_amount'])
-            ? $charge['refunded_amount']
-            : $charge['refunded'];
```

Comment:
I think `refunded` is used in API version 2014-2017, so we should still support both versions.

Reply:
@hpariwat Thanks for letting me know. I have add it again.